shared/scripts/lib/processors/dwolla.js

```diff
@@ -9,12 +9,6 @@ export function inject($el, amount, token) {
   var redirect = [root, base, 'Dwolla user', amount].join('/');
   var src = 'https://www.dwolla.com/scripts/button.min.js';
 
-  // If we're in testing, swap the token with our test account.
-  if (localStorage.testing === 'true') {
-    token = 'lYNGTjRZRQAU4j32+qB4fBAPNDTQQGeZHF9cZFrH+83qm21sTL';
-    src = base + '/../../vendor/dwolla.js';
-  }
-
```

Comment:
I'm not the original dev but I'm pretty sure this is as intended and is not a security issue. These tokens appear to be public identifiers and this code exists to avoid spamming legitimate Dwolla accounts when testing.

Unfortunately, I can't find any sites that actually use Dwolla and I'm not even sure if this API is still working...